README.md

```diff
@@ -32,7 +32,8 @@ The code for the Molecule Attention Transformer baseline is added as a submodule
 
 ## Data
 
-The actual dataset is stored in `dataset/`, split into `train`, `valid` and `test` folders.
+The dataset is available as a download, [FS-Mol Data](https://figshare.com/ndownloader/files/31345321), split into `train`, `valid` and `test` folders. Additionally, we specify which tasks are to be used with the file `dataset/fsmol-0.1.json`, a default list of tasks for each data fold. We note that the complete dataset contains many more tasks. Should use of all possible training tasks available be desired, the training script argument `--task_list_file` `dataset/entire_train_set.json` should be used. The task lists will be used to version FS-Mol in future iterations as more data becomes available via ChEMBL.
```

Comment:
The link works for me, and downloads a valid `*.tar` file with the directory structure as described 🙂

But, I'm assuming it should be
```suggestion
The dataset is available as a download, [FS-Mol Data](https://figshare.com/ndownloader/files/31345321), split into `train`, `valid` and `test` folders. Additionally, we specify which tasks are to be used with the file `datasets/fsmol-0.1.json`, a default list of tasks for each data fold. We note that the complete dataset contains many more tasks. Should use of all possible training tasks available be desired, the training script argument `--task_list_file` `datasets/entire_train_set.json` should be used. The task lists will be used to version FS-Mol in future iterations as more data becomes available via ChEMBL.
```

By the way, is "available as a download" a valid expression?